stochatreat/stochatreat.py

```diff
@@ -235,3 +243,7 @@ def stochatreat(data: pd.DataFrame,
     assert len(ids_treats) == len(data)
     assert ids_treats['treat'].isnull().sum() == 0
     return ids_treats
+
+
+def lcm(denominators):
+    return reduce(lambda a,b: a*b // gcd(a,b), denominators)
```

Comment:
Should that be `a*b // gcd(a,b)` or `a*b / gcd(a,b)`?

Reply:
Hum, I think `//` to make sure it returns an `int` (the other should return a `float` that can be safely cast to `int`)

Reply:
Ah! Fair enough.